com.gitlab.davem.ClamTk.yaml

```diff
@@ -120,13 +120,13 @@ modules:
         sitelib=$(perl -e 'use Config; print $Config{installsitelib};')
         install -d $sitelib
         cp -r lib $sitelib/ClamTk
-      - 'install -Dm 644 clamtk.desktop /app/share/applications/com.gitlab.davem.ClamTk.desktop'
+      - 'install -Dm 644 clamtk.desktop /app/share/applications/${FLATPAK_ID}.desktop'
       - 'install -Dm 644 images/clamtk.png /app/share/icons/hicolor/128x128/apps/clamtk.png'
-      - 'install -Dm 644 com.gitlab.davem.ClamTk.metainfo.xml -t /app/share/metainfo'
+      - 'install -Dm 644 com.github.davetheunsub.clamtk.appdata.xml /app/share/metainfo/${FLATPAK_ID}.appdata.xml'
```

Comment:
Any reason to rename metainfo -> appdata.xml?

Reply:
I'm not sure why it was `.metainfo` since upstream is `.appdata` https://github.com/dave-theunsub/clamtk/blob/master/com.github.davetheunsub.clamtk.appdata.xml

Reply:
Yea it doesn't work with metainfo

https://buildbot.flathub.org/#/builders/6/builds/33919

```
install: cannot stat 'com.gitlab.davem.ClamTk.metainfo.xml': No such file or directory
FB: host_command_exited_cb 932508 256
]2;flatpak-builder: Cleanup clamtkError: module clamtk: Child process exited with code 1
```

Reply:
Sorry I meant the *output* filename here, not the input one.